<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ethernet</class>
 <widget class="QMainWindow" name="Ethernet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Catena - Jaton - Olivera</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icono.ico</normaloff>Icono.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Conexión Ethernet</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Puerto:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="port">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startServer">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Arrancar Servidor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Estado:</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Conexión Serie</string>
    </property>
    <widget class="QPushButton" name="btn_conectar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Conectar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_desconectar">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Desconectar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Estado:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_estado">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Señal</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Valor Actual:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_val_act_sen">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="grfc_Sen">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>191</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_grfc_Sen">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpiar Gráfica</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_maxSen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_midSen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Luz</string>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Valor Actual:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_val_act_luz">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_grfc_Luz">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpiar Gráfica</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="grfc_Luz">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>191</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lab_maxLuz">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_midLuz">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
